tensorboard/backend/application.py

```diff
@@ -227,10 +227,15 @@ def _serve_plugins_listing(self, request):
     Returns:
       A werkzeug.Response object.
     """
-    return http_util.Respond(
-        request,
-        {plugin.plugin_name: plugin.is_active() for plugin in self._plugins},
-        'application/json')
+    response = {}
+    for plugin in self._plugins:
+      start = time.time()
+      response[plugin.plugin_name] = plugin.is_active()
+      elapsed = time.time() - start
+      tf.logging.info(
+          'Plugin listing: is_active() for %s took %0.3f seconds',
+          plugin.plugin_name, elapsed);
```

Comment:
nit: Looks like you got an extra semicolon here.

Reply:
Fixed, no idea how that crept in.